### PR TITLE
New responses aligned with current implementation

### PR DESCRIPTION
**Pull request by joselsegura**
# Description

Adapting the mock responses to the new data structure returned by the real service and add some more possible scenarios:

- No upgrade risks detected
- Upgrade risks detected
- Managed cluster
- AMS not available
- Upgrade Risks service unavailable

## Type of change

- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Refactor (refactoring code, removing useless files)
- Bump-up dependent library (no changes in the code)

## Testing steps

Run locally

## Checklist
* [ ] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: server/upgrade_risks_prediction.go [hunk 1]
```diff
@@ -43,6 +50,33 @@ func (server *HTTPServer) upgradeRisksPrediction(writer http.ResponseWriter, req
 		return
 	}
 
+	if clusterName == ClusterManaged {
```
Comment (tisnik) [inline server/upgrade_risks_prediction.go:53]: `switch` perhaps?

**Review by tisnik — COMMENTED** [review]
Probably pls use `switch`. LGTM instead

File: server/upgrade_risks_prediction.go (hunk 1 above)
Comment (joselsegura) [reply server/upgrade_risks_prediction.go]: You are right, fixed

File: server/upgrade_risks_prediction.go [hunk 2]
```diff
@@ -43,20 +55,49 @@ func (server *HTTPServer) upgradeRisksPrediction(writer http.ResponseWriter, req
...
-	if err != nil {
-		log.Error().Err(err).Msg(responseDataError)
+	case ClusterNoAMS:
+		log.Info().Msg("No AMS available case")
+		err = responses.SendServiceUnavailable(writer, "AMS service unavailable")
+		if err != nil {
+			log.Error().Err(err).Msg(responseDataError)
+		}
+
+	case ClusterUnavailable:
```
Comment (juandspy) [inline server/upgrade_risks_prediction.go:73]: is this duplicated from `ClusterNoAMS`?

Comment (joselsegura) [reply server/upgrade_risks_prediction.go:73]: The result is the same ATM, but can be different at any time. They are 2 different scenarios (AMS not available and Upgrade Risks Prediction not available), so I preferred to differentiate both

Comment (juandspy) [reply server/upgrade_risks_prediction.go:73]: Is the `Upgrade Risks Prediction not available` for the `ClusterUnavailable`? In that case the `log.Info` and `responses.SendServiceUnavailable(writer, "AMS service unavailable")` is duplicated from the previous one.

**Review by tisnik — APPROVED** [review]
seems legit

**Review by juandspy — APPROVED** [review]

**Review by juandspy — APPROVED** [review]

**Comment by joselsegura** [comment]
@tisnik could you please merge this one, or give me access to merge PRs by myself?

**Pull request merged**